etc/local_grapl/bin/upload-sysmon-logs.py

```diff
@@ -36,22 +35,6 @@ def hack_PATH_to_include_grapl_tests_common() -> Callable:
     return upload_sysmon_logs
 
 
-def setup_env(bucket_prefix: str):
```

Comment:
Have you manually tested that this script works?
(It's awful, but as it stands, it needs to work outside of any Docker context)

Reply:
@wimax-grapl Can you tell me a bit more about the intended usecases for this script? The `setup_env` function here only did anything when dealing with the `local-grapl` prefix, so using it against real AWS wouldn't be affected.

Are you saying that this script needs to be able to run against local Grapl from outside the docker-compose environment?